app/main.py

```diff
@@ -1,8 +1,31 @@
 class Person:
-    # write your code here
-    pass
+    people: dict[str, "Person"] = {}
```

Comment:
The type hint `dict[str, "Person"]` is correct, but ensure that the string 'Person' is necessary. In Python 3.9 and later, you can use `dict[str, Person]` without quotes if the class is defined before its usage.